src/lib/datepicker/datepicker.md

```diff
@@ -53,15 +53,16 @@ year containing the `startAt` date.
 
 #### Watching the views for changes on selected years and months
 
-When a year or a month is selected in `multi-year` and `year` views respecively, the `yearSelected`
+When a year or a month is selected in `multi-year` and `year` views respectively, the `yearSelected`
 and `monthSelected` outputs emit a normalized date representing the chosen year or month. By
 "normalized" we mean that the dates representing a year will have their month set to January and
-their day set to the 1st. Dates representing months will have their day set to the 1st of the
+their day set to the 1st. Dates representing a month will have their day set to the 1st of that
```

Comment:
I think that this one was correct to begin with.

Reply:
So I'll change the previous sentence from "a year" to "years" too.

Reply:
Oh, I got it.

Reply:
Just to confirm, it would be like this:

>... the dates representing years will have their month set to January and their day set to the 1st. Dates representing months will have their day set to the 1st of the ...

Reply:
Yes.